cmd/sblookup/main.go

```diff
@@ -42,6 +42,7 @@ var (
 	apiKeyFlag    = flag.String("apikey", "", "specify your Safe Browsing API key")
 	databaseFlag  = flag.String("db", "", "path to the Safe Browsing database. By default persistent storage is disabled (not recommended).")
 	serverURLFlag = flag.String("server", safebrowsing.DefaultServerURL, "Safebrowsing API server address.")
+	proxyFlag     = flag.String("proxy", os.Getenv("HTTP_PROXY"), "proxy to use to connect to the HTTP server")
```

Comment:
Do we need to have a non-empty default (for either binary)?  If it is empty, it will still respect $HTTP_PROXY (and $NO_PROXY)

Reply:
That's a good point, however having the default specified at that level increases clarity, in my opinion. Plus it ensures the behavior is understood by the user, just by reading `main.go` (instead of having to dig in `api.go`, and then `net/http`, should it be needed).
I guess it's more a question of user-experience and readability... Would you rather have it unspecified here, and let  `Transport` default settings kick in?

Reply:
I think it would be better to leave it out. The *_PROXY environment variables are fairly common (ex: wget and curl use them), it's not just a Go specific thing.

A side effect of this default is that NO_PROXY variables will be ignored (and NO_PROXY would most likely only be set when HTTP_PROXY was already set). So in order to get 'default' behavior, you'd have to say ./sblookup -proxy="", which would then use both env variables. That said, it is a fairly unlikely-to-trigger edge case.